packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -815,7 +823,7 @@ export class GarbageCollector implements IGarbageCollector {
         // If running in tombstone mode, initialize the tombstone state from the snapshot. Also, notify the runtime of
         // tombstone routes.
         if (this.tombstoneMode && baseSnapshotData.tombstones !== undefined) {
-            this.tombstones = Array.from(baseSnapshotData.tombstones);
+            this.tombstones = baseSnapshotData.tombstones;
```

Comment:
Why remove Array.from?

Reply:
It may be because I was manually reverting changes, do we need it?

Reply:
Yes, its important because it copies the array values rather than creating a reference to the array. The base snapshot may change later (I have seen something like this happening in other places) resulting in changing the tombstones list.

Reply:
The basic idea is to always create a copy of data you are writing from a source that you don't control.

Reply:
Got it, I'm going to put in a comment, so I don't accidentally revert it again